content/en/docs/concepts/security/service-accounts.md

```diff
@@ -155,6 +155,9 @@ To prevent Kubernetes from automatically injecting
 credentials for a specified ServiceAccount or the `default` ServiceAccount, set the
 `automountServiceAccountToken` field in your Pod specification to `false`.
 
+Service accounts and service account tokens bound to a pod become invalid 1
+minute after their `.metadata.deletionTimestamp`, regardless of finalizers.
```

Comment:
This is a bit confusing. You cannot set deletion timestamp on a bound token. Did you mean deletion timestamp is set on the pod?

Reply:
Hi @stlaz, yes. I updated according to https://github.com/kubernetes/website/pull/47404#discussion_r1711005225 and it may be clearer now.